src/config.inc

```diff
@@ -76,7 +76,7 @@ endif
 OSX_IDENTITY="Developer ID Application: Daniel Kroening"
 
 # Detailed version information
-CBMC_VERSION = 5.63.0
+CBMC_VERSION = 5.64.0
```

Comment:
Please remove this change.

It conflicts with the current version of `src/config.inc`, and releases is something we do on a specific schedule, unless there's an emergency of sorts that requires a hot fix.